freqtrade/rpc/telegram.py

```diff
@@ -1677,3 +1678,26 @@ def _send_msg(self, msg: str, parse_mode: str = ParseMode.MARKDOWN,
                 'TelegramError: %s! Giving up on that message.',
                 telegram_err.message
             )
+
+    @authorized_only
+    def _changemarketdir(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /marketdir.
+        Updates the bot's market_direction
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+        if context.args and len(context.args) == 1:
+            new_market_dir = context.args[0]
+            match new_market_dir:
+                case "long":
+                    self._rpc._freqtrade.strategy.market_direction = MarketDirection.LONG
```

Comment:
conceptually, i'd probably move the actual assignment to the rpc class (which would be mostly aligned how other methods in this regards are setup).

That'll make this (eventually) available easily for other rpc methods (namely, the API) - should we decide to add support for that there as well.
Not a hard requirement - but adding this since you asked :)